GoogleForCreators/web-stories-wp
# Reduced delay before displaying clone when dragging from library

## Context

What is says in the title.

## User-facing changes

Notice the delay before the clone shows before and after:

| Before | After |
|-|-|
| ![drag-slow-before](https://user-images.githubusercontent.com/637548/117224323-a276bb80-addd-11eb-9cef-5c9a6682a838.gif) | ![drag-slow-after](https://user-images.githubusercontent.com/637548/117224324-a3a7e880-addd-11eb-9e7a-0e09e10c2609.gif) |

Notice what happens for a quick drag-and-hold slightly off the original location before and after as well:

| Before | After |
|-|-|
| ![drag-fast-before](https://user-images.githubusercontent.com/637548/117224340-ac98ba00-addd-11eb-905a-726e6f801122.gif) | ![drag-fast-after](https://user-images.githubusercontent.com/637548/117224345-ae627d80-addd-11eb-87f2-d30c17936856.gif) |

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Drag an image from the local library and observe how quickly the dragged clone appears (much faster than before - 50ms versus 300ms).

### UAT




## Checklist




---

Fixes #6027
